crates/networking/discv5/src/discovery.rs

```diff
@@ -329,7 +334,22 @@ impl NetworkBehaviour for Discovery {
                     }
                 }
             }
-            EventStream::Present(_receiver) => {}
+            EventStream::Present(receiver) => match receiver.try_recv() {
+                Ok(event) => {
+                    if let Event::SocketUpdated(socket_addr) = event {
+                        self.discv5.update_local_enr_socket(socket_addr, true);
+                        if let Some(enr_sender) = &self.enr_update_sender {
+                            if let Err(e) = enr_sender.send(self.discv5.local_enr()) {
+                                warn!("Failed to send socket address update: {:?}", e);
+                            }
+                        }
+                    }
+                }
+                Err(error) => {
+                    error!("Failed to start discovery event stream: {:?}", error);
+                    self.event_stream = EventStream::Inactive;
```

Comment:
We should stay consistent with our error variable names, could you use `err` instead of `error` and `e` and inline them {err:?}